django_project/cplus_api/tasks/remove_layers.py

```diff
@@ -0,0 +1,50 @@
+import logging
+from datetime import timedelta
+from celery import shared_task
+from django.utils import timezone
+
+from cplus_api.models import InputLayer, OutputLayer, UserProfile
+
+logger = logging.getLogger(__name__)
+
+
+@shared_task(name="remove_layers")
+def remove_layers():
+    """
+    Remove layer that has been more than 2 weeks.
+    """
+    ext_users = UserProfile.objects.exclude(
```

Comment:
@zamuzakki we don't need to filter the layers by ext_users. Below is the logic for removing layers:

- [ ] InputLayer: remove 'private' layer that is older than 14 days
- [ ] OutputLayer: remove layers other than final_output layer and group='weighted_ims'

Reply:
Please add output_group_to_keep in the site preferences to store the 'weighted_ims'